packages/schemas/extract/src/git-identities.ts

```diff
@@ -0,0 +1,15 @@
+import type { InferModel } from "drizzle-orm";
+import { sql } from 'drizzle-orm';
+import { sqliteTable, text, integer, primaryKey } from 'drizzle-orm/sqlite-core';
+
+export const gitIdentities = sqliteTable('git_identities', {
+    memberId: integer('member_id').notNull(),
+    authorEmail: text('author_email').notNull(), // -> ToDo -> check for adding repo_id
```

Comment:
Why is there an author_email? These should be git identities defined by name + email.